builtin/provisioners/habitat/resource_provisioner_test.go

```diff
@@ -43,13 +44,14 @@ func TestResourceProvisioner_Validate_bad(t *testing.T) {
 	if len(warn) > 0 {
 		t.Fatalf("Warnings: %v", warn)
 	}
-	if len(errs) != 1 {
-		t.Fatalf("Should have one error")
+	if len(errs) != 2 {
+		t.Fatalf("Should have two errors")
```

Comment:
Curious question (I see there's not info about the original error) but what was the one error, and what are the two? (a comment might be nice here, maybe it's elsewhere and I haven't caught it)

Reply:
This test produced one error before accept_license flag was introduced. This error is "invalidtype is not a valid service_type." However since we have introduced accept_license as a required argument, now we get two errors in this test. The second one being: "accept_license": required field is not set. I can add a comment.